zipline/utils/numpy_utils.py

```diff
@@ -321,7 +324,7 @@ def rolling_window(array, length):
 
 # Sentinel value that isn't NaT.
 _notNaT = make_datetime64D(0)
-iNaT = NaTns.view(int64_dtype)
+iNaT = int(NaTns.view(int64_dtype))
```

Comment:
I would have expected this to be an int64. What's the reason for converting to a python int here?

Reply:
The only usage of this now (it was unused before, except in the assert below) is to represent a missing `first_trading_day` attribute in the bcolz ctable for us equity pricing, which requires json-serializability. We used to use `pandas.tslib.iNaT` (a [`long`](https://github.com/numpy/numpy/blob/master/numpy/core/include/numpy/npy_common.h#L420-L421)) for this.